src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        UserService userService = new UserService();
+
+        if (userService.findByEmail(email).getPassword().equals(password)) {
+            return true;
+        }
+
```

Comment:
```suggestion
        UserService userService = new UserService();
        if (userService.findByEmail(email).getPassword().equals(password)) {
            return true;
        }
```
Let’s remove redundant empty lines. 
Can method `findByEmail(email)` return null? Can you get NPE?